src/App.css

```diff
@@ -48,14 +48,158 @@ input {
   box-shadow: 1px 1px 2px black;
 }
 
-button {
+.homeWrap {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-color: lightblue;
+  height: 50vh;
+  width: 30vw;
+  justify-content: space-around;
+
+}
+
+.joinLink {
+  text-decoration: none;
+  color: black;
+  margin: 3vh;
+  padding: 1vh;
+  border: 2px solid black;
+  border-radius: 2px;
+  background-color: white;
+  font-size: 25px;
+}
+
+.joinLink:hover {
+  font-size: 26px;
+  box-shadow: 1px 1px 2px black;
+
+}
+
+.newListLink {
+  color: black;
+  text-decoration: none;
+  border: 2px solid black;
+  margin: 1vh;
+  border-radius: 2px;
+  background-color: white;
+  padding: 1vh;
+
+}
+
+.newListLink:hover {
+  box-shadow: 1px 1px 2px black;
+}
+
+.joinListButton {
+  color: black;
+}
+
+.tokenButton {
+  position: relative;
   background-color: white;
   border: 2px solid black;
   padding: 1vh;
   margin: 1vh;
+  height: 10vh;
+  border-radius: 2px;
 }
 
-button:hover {
-  font-size: 23px;
+.tokenButton:hover {
+  font-size: 32px;
   box-shadow: 1px 1px 2px black;
+  transition: .1s;
+  transition-timing-function: ease-in;
+}
+
+.addForm {
+  /* display: flex; */
+  border: 1px solid black;
+  justify-content: center;
+  align-items: center;
+  width: 60vw;
+  padding: 2vh;
+  height: 50vh;
+  background-color: lightblue;
+  /* position: absolute; */
+}
+
+.addItemButton {
+  /* display: flex; */
+  background-color: white;
+  border: 2px solid black;
+  padding: 1vh;
+  margin: 1vh;
+  height: 10vh;
+  /* align-self: flex-end; */
+  /* position: relative; */
+}
+
+.addItemButton:hover {
+  font-size: 32px;
+  box-shadow: 1px 1px 2px black;
+  transition: .1s;
+  transition-timing-function: ease-in;
+}
+
+.itemInputField {
```

Comment:
![image](https://user-images.githubusercontent.com/4306/69909554-4c644400-13b1-11ea-94fc-56fcd721bd63.png)

Non-blocking because it does not affect functionality, but the input field blows out past it's container on mobile

Reply:
(The above is Microsoft Edge with the web inspector open and the device view set to mobile)